helper/builtinplugins/registry.go

```diff
@@ -147,15 +146,11 @@ func newRegistry() *registry {
 				Factory:           logicalCass.Factory,
 				DeprecationStatus: consts.PendingRemoval,
 			},
-			"consul":     {Factory: logicalConsul.Factory},
-			"gcp":        {Factory: logicalGcp.Factory},
-			"gcpkms":     {Factory: logicalGcpKms.Factory},
-			"kubernetes": {Factory: logicalKube.Factory},
-			"kv":         {Factory: logicalKv.Factory},
-			"mongodb": {
-				Factory:           logicalMongo.Factory,
-				DeprecationStatus: consts.PendingRemoval,
```

Comment:
I'm wondering if the right thing to do is move this over to `consts.Removed`. I will check on this to find out.

Reply:
Yeah I think `consts.Removed` is correct. Removing this engine seems fine to me, but should probably be separated into its own PR (with a `change` type changelog). The fix part further down this PR may want to be backported, whereas this removal definitely needs to wait for 1.13.0, so having two separate PRs just lets you control the lifecycle of each change more easily.

Reply:
+1 @tomhjp - We should separate the removal of the engines into a different PR.